spec/controllers/import/github_controller_spec.rb

```diff
@@ -10,13 +10,16 @@
   describe "GET callback" do
     it "updates access token" do
       token = "asdasd12345"
-      Gitlab::GithubImport::Client.any_instance.stub(:get_token).and_return(token)
-      Gitlab.config.omniauth.providers << OpenStruct.new(app_id: "asd123", app_secret: "asd123", name: "github")
+      allow_any_instance_of(Gitlab::GithubImport::Client).
+        to receive(:get_token).and_return(token)
+      Gitlab.config.omniauth.providers << OpenStruct.new(app_id: "asd123",
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.